pkgs/tools/misc/ollama/default.nix

```diff
@@ -1,36 +1,55 @@
 { lib
+, stdenv
+, fetchpatch
 , buildGoModule
 , fetchFromGitHub
 , llama-cpp
+, darwin
 }:
 
-buildGoModule rec {
+let
+  version = "0.1.25";
+  llama_lib = (llama-cpp.override {
+      static = false;
+    }).overrideDerivation (oldAttrs: {
```

Comment:
We *could* also override the version here to match ollama's vendoring. Things seem to work though. I'm happy to go either way.